compiler/compile/Method.cpp

```diff
@@ -351,6 +352,7 @@ TR_Method::isBigDecimalConvertersMethod(TR::Compilation * comp)
    return false;
    }
 
+TR::IlType * TR_ResolvedMethod::getParmIlType(int32_t parmIndex)           { TR_UNIMPLEMENTED(); return 0; }
```

Comment:
I didn't like it either.  I rewrote this to use the existing `parmType` function, and introduced a new `getParameterTypeSignature` function that returns a `char *` rather than exposing `TR::IlType` publicly.